doc/lua_api.md

```diff
@@ -8802,6 +8802,10 @@ child will follow movement and rotation of that bone.
             but keeping original luma and being symmetrical in terms of saturation
             (eg. -1 and 1 is the same saturation and luma, but different hues)
         * This value has no effect on clients who have shaders or post-processing disabled.
+      * `foliage_translucency` controls the intensity of the foliage translucency effect (default: `1.5`).
```

Comment:
How and where can I see this parameter in effect?

`minetest.conf` settings (just in case):
```
enable_post_processing = true
enable_dynamic_shadows = true
shadow_strength_gamma = 0.5
enable_color_grading = true
enable_volumetric_lighting = true
enable_volumetric_depth_attenuation = true
enable_sun_tint = true
enable_bloom = true
enable_translucent_foliage = true
```

Test commands (WorldEdit): 
```
//lua player:set_lighting({ foliage_translucency = 4.0 })
//lua player:set_lighting({ foliage_translucency = 0 })
```